srml/staking/src/inflation.rs

```diff
@@ -64,7 +72,7 @@ impl PiecewiseLinear {
 	}
 }
 
-// Piecewise linear approximation of I_NPoS.
+/// Piecewise linear approximation of I_NPoS.
```

Comment:
```suggestion
/// Piecewise linear approximation of `I_NPoS`.
```

Also, there needs to be documentation of what `I_NPoS` means.  Users should not need to have read any research papers to understand the code.

Reply:
hmm this comment is internal comments as const is private, the only public function of the module is `compute_total_payout` and should be understandable.

Maybe I should organize differently with a top documentation

Reply:
@thiolliere That works too.  I was referring to people reading the source code, not consumers of it.